source/extensions/filters/http/lua/wrappers.cc

```diff
@@ -41,8 +41,8 @@ int HeaderMapWrapper::luaAdd(lua_State* state) {
 }
 
 int HeaderMapWrapper::luaGet(lua_State* state) {
-  const char* key = luaL_checkstring(state, 2);
-  const auto value =
+  const absl::string_view key = Filters::Common::Lua::getStringViewFromLuaString(state, 2);
```

Comment:
Nit. no need to put const on `absl::string_view`.

Reply:
Removed